templates/checkout/login.html

```diff
@@ -10,7 +10,7 @@
       <div class="checkout__login__continue">
         <h3>{% trans "New customer?" context "Checkout login page title" %}</h3>
         <img class="checkout-pirate" src="{% static 'images/pirate_login.png' %}"
-             srcset="{% static 'images/pirate-checkout.png' %} 1x, {% static 'images/pirate-checkout2x.png' %} 2x">
```

Comment:
They are some other places, where this image is used, I think we should:
1. Rename this file
2. Update the name in all relevant places

Reply:
Well, yes, a lot of static files and templates use dashes and underscores interchangeably which should be obviously unified. We should open a new issue for that and do it in one batch in a new PR.